configure.ac

```diff
@@ -27,7 +27,7 @@ AC_CONFIG_FILES([stamp-h], [echo timestamp > stamp-h])
 
 # sm: require a late-enough autoconf; this is the version number
 # that's on manju, so I assume it's ok
-AC_PREREQ([2.69])
+AC_PREREQ([2.71])
```

Comment:
Do we really need `2.71`?
All `Ubuntu`s up to and including `Ubuntu 21.04` only have `2.69` available from default package sources.

Reply:
I don't know, it was generated like that.
Just tried it on the server with Ubuntu 18.04.5 LTS and autoconf 2.69-11 and it seems to be fine:
```
------------ [408] CC=gcc make testrungcc/enum3l ------------

Successful tests:     374
Failed as expected:   31
Unexpected success:   4
Unexpected failure:   0
```

There it also complained about `cl`, so that's nothing specific to my setup:
```
Generating machine dependency information for CIL
/bin/sh: 1: cl: not found
```
Looks like it's for MSVC, so I don't know why it's trying to call it with `HAS_MSVC` correctly being identified as `no` before.

Reply:
> Just tried it on the server with Ubuntu 18.04.5 LTS and autoconf 2.69-11 and it seems to be fine:

It will run fine, but one is unable to run `autoconf` to generate the `configure` file on these machines then. Does it still work if you set `AC_PREREQ([2.69])` on your `M1` machine? Or alternatively, generate the files on Ubuntu and then use them on your machine?

Reply:
> Or alternatively, generate the files on Ubuntu and then use them on your machine?

No, generating it again on Ubuntu with 2.69 does not add the needed `arm64`.

> Does it still work if you set AC_PREREQ([2.69]) on your M1 machine?

Yes, but autoconf 2.69 then complains:

~~~console
❯ autoconf
configure.ac:176: error: possibly undefined macro: AC_CHECK_INCLUDES_DEFAULT
      If this token and others are legitimate, please use m4_pattern_allow.
      See the Autoconf documentation.
~~~

Since `configure` is committed and self-contained shell script, it's only relevant for developers.
It seems like we only regenerate `configure` when there's some new gcc-n added on macOS (and when CIL's version changes but maybe that's not necessary?).
How hard is it to run autoconf 2.71 on Ubuntu?